docs/spec/metrics/sample.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "docs/spec/metrics/sample.json",
+    "type": [
+        "object"
+    ],
+    "description": "A single metric sample.",
+    "oneOf": [
+        {
+            "additionalProperties": false,
+            "properties": {
+                "type": {"enum": ["counter"]},
+                "unit": {"type": "string"},
+                "count": {"type": "number"}
```

Comment:
fwiw i also vote for calling this one `value` too

Reply:
@roncohen thoughts?  I don't feel strongly either way and this would be a quick update.